pkg/addons/data/Deployment.apps/openshift-monitoring/mdm.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: mdm-shard
+  name: mdm
+  namespace: openshift-monitoring
```

Comment:
We running in Prometheus namespace. Not nice :/
We need this because statsD calls prometheus endpoints directly. And we do not allow inter-project communication if we run this in separate namespace